src/_locales/de-DE.json

```diff
@@ -0,0 +1,94 @@
+{
+  "audio-device-communications": "System Default Communications",
+  "audio-device-default": "Standardeinstellung",
+  "audio-device-unknown": "Unbekanntes Gerät",
```

Comment:
I think, in ›Standardkommunikationsgerät‹ should be a ›system‹ anywhere to show the user where the settings are coming from..